yjit.rb

```diff
@@ -168,12 +180,22 @@ def self.simulate_oom!
 
   # Avoid calling a method here to not interfere with compilation tests
   if Primitive.rb_yjit_stats_enabled_p
-    at_exit { _print_stats }
+    at_exit do
+      _print_stats
+      _dump_locations
```

Comment:
I think this should not be called unless `trace_exit_locations_enabled?` because it seems to raise ArgumentError when only `--yjit-stats` is given.

Reply:
Oh right, yes I will update.

Reply:
Fixed now 😅